tests/assets/SPECS/rpm-file-attrs.spec

```diff
@@ -0,0 +1,68 @@
+Name:           rpm-file-attrs
+Version:        1.0
+Release:        1
+Summary:        Test RPM file attributes
+License:        MIT
+BuildArch:	    noarch
+
+%description
+%{summary}
+
+%install
+mkdir -p %{buildroot}/opt/%{name}
+
+echo "normal" > %{buildroot}/opt/%{name}/normal
+mkdir -p %{buildroot}/opt/%{name}/dir/
+echo "file-in-a-dir" > %{buildroot}/opt/%{name}/dir/normal
+
+# symlinks to a file and directory
+mkdir -p %{buildroot}/opt/%{name}/symlink_dir/
+ln -s ./normal %{buildroot}/opt/%{name}/symlink
+ln -s ../dir %{buildroot}/opt/%{name}/symlink_dir/
```

Comment:
To add a bit more specificity to what was going wrong earlier,

without the "mkdir" it yields an error "No such file or directory" if there is a trailing slash, or "Not a directory" if there is not.  With the "mkdir" it succeeds but the symlink is placed inside of "symlink_dir" instead of at that path.

I still find this confusing, because when I create that symlink manually without "mkdir" it works fine.  Only inside of rpm does it seem to fail.

Reply:
Could you add that as an information nugget, it will be lost if it's only in the PR